src/plugin.js

```diff
@@ -10,6 +13,9 @@ function initPlugin() {
     reader.onload = () => {
       const base64Data = reader.result;
```

Comment:
Are we sure this always has some value in it?

Reply:
From what I can tell based on the `onload` literature, yes.